### ISSUE TITLE
Reading "id" attribute

### ISSUE THREAD

**Issue by r0hitsharma** [balderdashy/sails-mongo#182]
Is it possible to read `id` attribute and not have it overridden by the default `_id` field in mongo being interpreted as `id` by sails-mongo?

**Comment by ksaitor** [comment]
Having the same issue. Any luck here?

**Comment by juanitomint** [comment]
Same to me I already have documents in the DB with "id" field name

**Comment by akshitjain** [comment]
Same issue occurring making it difficult to link the database in mongodb and mysql as mysql makes 'id' as attribute by default while mongo creates _id by default.While inserting the data we can specify 'id' only and it works but while querying it differentiates between 'id' and '_id'.

**Comment by sailsbot** [comment]
Thanks for posting, @r0hitsharma.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by juanitomint** [comment]
damn you sailsbot! hahahahah
